superset/views/sql_lab.py

```diff
@@ -162,6 +162,7 @@ class SavedQueryViewApi(SavedQueryView):  # pylint: disable=too-many-ancestors
         "description",
         "sql",
         "extra_json",
+        "extra",
```

Comment:
I'm surprised to see both `extra` and `extra_json` - what's the difference/motivation?

Reply:
`extra_json` is a string containing the serialized JSON, `extra` is the actual nested data structure. While I know the private API does not use `extra_json`, I assume that people in the wild (notably at Lyft where @betodealmeida shipped the feature) are very likely to use it. 

Clearly it's not great to have JSON inception (serialized JSON inside of JSON)

I could have deprecated `extra_json` in favor of `extra`, but would need to do change management around it - likely just adding a note in `UPDATING.md`. Eventually we should move this under `/api/v1` but, maybe a good time to do this.